filament/backend/src/metal/MetalEnums.h

```diff
@@ -435,6 +435,21 @@ inline MTLTextureSwizzleChannels getSwizzleChannels(TextureSwizzle r, TextureSwi
             getSwizzle(a));
 }
 
+inline bool isStencilFormat(MTLPixelFormat format) {
```

Comment:
nit: maybe `formatHasStencil()` name would indicate more clearly that this function isn't just about stencil-only formats